packages/devtools/lib/src/memory/memory.dart

```diff
@@ -1082,7 +1107,9 @@ class MemoryScreen extends Screen with SetStateMixin {
 
       final InstanceRef ref = value;
 
-      if (ref.valueAsString != null && !ref.valueAsStringIsTruncated) {
+      if (ref != null &&
+          ref.valueAsString != null &&
```

Comment:
could combine these two to be `ref?.valueAsString != null`

Reply:
Done.